website/docs/writing-docs.mdx

```diff
@@ -0,0 +1,176 @@
+---
+id: writing-docs
+title: Writing Docs
+---
+
+<!-- TODO: The title is a bit weird -->
+
+Next, let's touch on the powerful feature in Docusaurus - documentation.
+
+## Using Markdown
+
+Docusaurus supports Markdown with [Remark](https://github.com/remarkjs/remark) as the Markdown parser and is extensible via plugins.
+
+<!-- TODO: Remark plugins? https://github.com/remarkjs/remark/blob/master/doc/plugins.md#list-of-plugins -->
+
+In the beginning of each of your docs, specify the `id`, the `title`, and `sidebar_label` so that Docusaurus will pick them up correctly when generating your site. We'll talk about that again in a minute.
+
+```markdown
+---
+id: greeting
+title: Hello
+sidebar_label: hello
+---
+
+# Hello from Docusaurus
+
+Are you ready to create the documentation site for your open source project?
+```
+
+<!-- TODO: talk about where to put the docs, resolving docs outside of website directory, etc. -->
+
+## Embedding React Components
+
+Docusaurus has in-house support for [MDX](https://mdxjs.com). This allows you to write JSX within your Markdown files and render them as React components.
+
+Rename your file from `.md` to `.mdx`. Docusaurus will hot reload your dev server and you won't need to restart.
```

Comment:
Why do we need `mdx` extension here. It can give an impression that it won't work without the `.x` extension. Let's stick with `.md`.

Reply:
Good point, will remove this.